website/questmaker/tests/excursion.sql

```diff
@@ -13,7 +13,7 @@ INSERT INTO quests (title, author_id, description, keyword, password, time_open,
 
 --places info
 --branch "Temple architecture"
-INSERT INTO places (coords, radius) VALUES ('(59.8633917835336, 30.472641018135985)', 50);
+INSERT INTO places (coords, radius) VALUES ( '(59.8633917835336, 30.472641018135985)', 50);
```

Comment:
For what? Why?
```suggestion
INSERT INTO places (coords, radius) VALUES ('(59.8633917835336, 30.472641018135985)', 50);
```

Reply:
![image](https://user-images.githubusercontent.com/55149826/167253891-589ec31a-ae81-4cc5-902e-766d7cbcee88.png)
For what?